test/test_init.rb

```diff
@@ -7,6 +7,7 @@
 
 require 'test_bench'; TestBench.activate
 
+require 'pry'
```

Comment:
we can probably remove this dev dep as part of this MR (and the gemspec changes)